addons/point_of_sale/static/src/js/devices.js

```diff
@@ -480,11 +480,24 @@ var ProxyDevice  = core.Class.extend(mixins.PropertiesMixin,{
         send_printing_job();
     },
 
-    print_sale_details: function() {
+    /** Print an overview of todays sales.
+     *
+     * By default this will print all sales of the day for current PoS config.
+     * 
+     * @param {boolean} current_session: If set, only sales for the current session will be printed.
+     */
+    print_sale_details: function(current_session) {
```

Comment:
no need to set argument current_session as called only once and you manually set true